Source/WebCore/rendering/GridMasonryLayout.h

```diff
@@ -41,8 +41,14 @@ class GridMasonryLayout {
     {
     }
 
+    enum class OverrideLogicalWidth {
```

Comment:
Not 100% sure that `OverrideLogicalWidth` is a great name for this enum. Currently it implies that if you pass this into masonry layout some sort of logical width but not sure for what. Maybe `OVerrideMasonryLogicalWidth`?

Reply:
Sounds good. Updated.